card/lib/card/migration.rb

```diff
@@ -28,8 +27,12 @@ def find_unused_name base_name
       test_name
     end
 
-    def paths mig_type=type
-      Card.paths["db/migrate#{schema_suffix type}"].to_a
+    def migration_paths mig_type=type
+      Cardio.migration_paths mig_type
+    end
```

Comment:
I'm merging this, but should we consider making it a rails paths object?  It's basically a paths object converted to an array, no?